portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,15 +23,36 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that sends and receives comments data. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private ArrayList<String> data = new ArrayList(Arrays.asList("Message 0", "This is message 1.", "MESSAGE TWO"));
+  private ArrayList<String> comments = new ArrayList();
```

Comment:
We probably want to declare this in doGet()

See https://google.github.io/styleguide/javaguide.html#s4.8.2-variable-declarations 

"local variables are declared close to the point they are first used (within reason), to minimize their scope"

Reply:
Looks like you already addressed this in the next pull request.